### PR TITLE
add tooltips on the library selectors

### LINKED ISSUE #1078: Tooltips for Libraries to add to build path
In the new project wizards, each individual library we can add to the build path such as Objectify and App Engine API, should have a tooltip explaining what this library is.

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh fix #1078

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.ui.test/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroupTest.java [hunk 1]
```diff
@@ -63,6 +63,13 @@ public void run() {
       }
     });
   }
+  
+  @Test
+  public void testTooltips() {
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.ui.test/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroupTest.java]: Tool_T_ips?

File: plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties [hunk 2]
```diff
@@ -13,4 +13,7 @@ cloud.sdk.not.installed=Could not find the Google Cloud SDK
 cloud.sdk.out.of.date=Cloud SDK too old
 appengine.java.component.missing=App Engine Java Component is missing
 appengine.java.component.not.installed=The Cloud SDK App Engine Java component is not installed
-appengine.libraries.group=Libraries to add to build path
\ No newline at end of file
+appengine.libraries.group=Libraries to add to build path
+objectify.tooltip=com.googlecode.objectify, a Java data access API designed for Google Cloud Datastore
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties]: Put `com.googlecode.objectify` at the end with in parentheses (like the `endpoints.tooltip` below)?

File: plugins/com.google.cloud.tools.eclipse.appengine.ui.test/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroupTest.java (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.ui.test/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineLibrariesSelectorGroupTest.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/messages.properties]: I sort of want a proper name that isn't redundant here.

**Pull request merged**